### DISCUSSION TITLE
Table transformer Sql

### DISCUSSION THREAD

**Discussion by Technewbie23** [agershun/alasql#1741] [Q&A]
hi,

 i am new to sql /programming, i am trying to transpose a table  . i  am looking to turn the values in a row into columns but they need to be grouped into a column based on the first 3 characters of the word

I used this query and I got an error that there are no rows to be returned . please help if you have any solution for me . Thank you

```
SELECT *
  CASE
   WHEN SUBSTRING([column], 1, 4) = 'ABC' THEN [COLUMN]
   ELSE NULL
   END AS COLUMN,
   
   FROM T*;
  ```

**Comment by mathiasrw** [comment]
Question about how to ...
- Please use http://stackoverflow.com/questions/ask?tags=alasql
